python/tvm/relay/frontend/pytorch.py

```diff
@@ -3254,10 +3259,14 @@ def from_pytorch(script_module, input_shapes, custom_convert_map=None, default_d
         TorchScripted PyTorch graph
         Note: We currently only support traces (ie: torch.jit.trace(model, input))
 
-    input_shapes : List of tuples of input name and input dimensions
-        Graph level input shape list
+    input_infos: List of tuples of input name and (input dimensions, input types)
+        Graph level input shape and type list
         The same input names need to be used for deployment, so choose easy to
         remember names (such as: input0, input1)
+        e.g. [('input0', ((1, 2), 'int')), ('input1', ((3, 4), 'float'))]
+        supported data types: ['half', 'float', 'double', 'bool', 'char', 
+                               'byte', 'short', 'int', 'long',
+                               'quint8', 'qint8', 'qint32', 'str']
```

Comment:
I don't think we support that many types.

Reply:
https://github.com/apache/incubator-tvm/blob/master/python/tvm/relay/frontend/pytorch.py#L2416 from here

Reply:
remove the comment